packages/api/package.json

```diff
@@ -33,6 +33,7 @@
     "form-data": "^3.0.0",
     "get-stream": "^6.0.0",
     "js-yaml": "^3.14.0",
+    "make-dir": "^3.1.0",
```

Comment:
This module is already coming in via `find-cache-dir`, so I'm using that instead of loading in something else like [mktemp](https://www.npmjs.com/package/mktemp).